packages/styles/stepper.css

```diff
@@ -116,3 +138,11 @@
   background: var(--accent-info);
   color: var(--accent-dark);
 }
+
+.cauldron--theme-dark .Stepper__step--current .Stepper__step-label {
+  color: var(--white);
+}
+
+.cauldron--theme-dark .Stepper__step--complete .Stepper__step-indicator svg {
+  color: var(--black);
+}
```

Comment:
A couple of thoughts here:

This is an older component, so some of the "patterns" here are a little bit outdated. Our guidelines are [here](https://cauldron.dequelabs.com/style-guidelines#theming) if you need a refresher, but we should generally lean towards avoiding long selectors as it makes it harder to style in an app and instead lean on css variables to set the colors according to the theme according to the current scope of the theme.

Additionally, `--black` doesn't exist and `--white` is primarily there for legacy purposes so `#fff` and `#000` are fine to set as the colors since those definitions won't be changing.